src/openrct2/object/Object.cpp

```diff
@@ -116,6 +116,16 @@ void rct_object_entry::SetName(const std::string_view& value)
     std::memcpy(name, value.data(), std::min(sizeof(name), value.size()));
 }
 
+const std::vector<std::string>& Object::GetAuthors() const
+{
+    return _authors;
+}
+
+void Object::SetAuthors(const std::vector<std::string>& authors)
```

Comment:
I still think you could make this more efficient using an r value reference. It would avoid one extra duplication of the vector

Reply:
Sorry, I had to read up on rvalues. To be clear does that just mean changing the argument signature to && and changing the set call to this (ie adding std::move)

`result->SetAuthors(std::move(authorVector));`

Reply:
Yup that would be all that is involved